### ISSUE TITLE
individuals with patriarchs: empty list for vowel (Umlaut) in surname

### ISSUE THREAD

**Issue by arbor95** [vesta-webtrees-2-custom-modules/vesta_extended_relationships#103]
individuals with vowel (Umlaut) in surname result in empty list view, if language is german.
you can try with:
https://freris.de/tree/frese/individual-patriarchs-list?surname=Füchten#

**Comment by ric2016** [comment]
The Vesta code for this list is based on webtrees code, which is rather hard to follow, and may have bugs itself.
I rather regret having offered this feature in the first place.

Anyway, I have fixed this in latest commit, hopefully without breaking anything else.
